dbms/tests/performance/ngram_distance.xml

```diff
@@ -25,22 +25,32 @@
         </any_of>
     </stop_conditions>
 
-    <query>SELECT DISTINCT URL,Title, ngramDistance(Title, URL) AS distance FROM hits_10m_single FORMAT Null</query>
-    <query>SELECT DISTINCT SearchPhrase,Title, ngramDistance(Title, SearchPhrase) AS distance FROM hits_10m_single FORMAT Null</query>
+    <substitutions>
+        <substitution>
+           <name>small_table</name>
+           <values>
+               <value>(select * from hits_10m_single limit 2000000)</value>
```

Comment:
If the query is executed with more than a single thread, the result is non-deterministic.